plugins/english/novelupdates.ts

```diff
@@ -142,8 +142,19 @@ class NovelUpdates implements Plugin.PluginBase {
 
     loadedCheerio = parseHTML(text);
 
+    const nameReplacements: { [key: string]: string } = {
+      'v': 'volume ',
+      'c': ' chapter ',
+      'part': 'part ',
+      'ss': 'SS',
+    };
+
     loadedCheerio('li.sp_li_chp').each((i, el) => {
-      const chapterName = loadedCheerio(el).text().trim();
+      let chapterName = loadedCheerio(el).text();
+      for (let name in nameReplacements) {
+        chapterName = chapterName.replace(name, nameReplacements[name]);
+      }
+      chapterName = chapterName.replace(/\b\w/g, l => l.toUpperCase()).trim();
```

Comment:
We could also capitalise `nameReplacements` instead of replacing again, right?

Reply:
Since I also want to capitalise words like `prologue` or `epilogue` without having to add every single possible word to `nameReplacements`, I wanted to first the necessary stuff like `v` to `volume ` or `c` to ` chapter ` and after that has been done, go over the whole chapterName and capitalise everything, not just words I have set as `nameReplacements`.